playground/PostgresEndToEnd/PostgresEndToEnd.AppHost/aspire-manifest.json

```diff
@@ -1,9 +1,10 @@
 {
+  "$schema": "https://json.schemastore.org/aspire-8.0.json",
   "resources": {
     "pg1": {
       "type": "container.v0",
       "connectionString": "Host={pg1.bindings.tcp.host};Port={pg1.bindings.tcp.port};Username=postgres;Password={pg1-password.value}",
-      "image": "postgres:16.2",
+      "image": "docker.io/library/postgres:16.2",
```

Comment:
Unrelated?

Reply:
Consequences of some other updates explicitly specifying registry use (to enable Podman). This manifest had not been regenerated since that change was made.